### ISSUE TITLE
About loss function

### ISSUE THREAD

**Issue by Prtick** [huawei-noah/Pretrained-IPT#54]
Hello, author

Thanks for your work!
In paper, I noticed that you used a contrastive loss function in your work, but I did not find relevant implementation in the code. I am very interested in this part and would like to ask if you could upload the relevant code snippets. Thank you very much!
